### PR TITLE
Add openSUSE Leap 15.6 Images

### PR DESCRIPTION
**Pull request by Vogtinator**

### CODE REVIEW

File: job_groups/opensuse_leap_15.6_images.yaml [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+          machine: 64bit
+          settings:
+            QEMURAM: "2048"
+            QEMU_VIRTIO_RNG: "0"
+      - kde_live_upgrade_leap_15.2:
+          machine: uefi
+          settings:
+            QEMURAM: "2048"
+            QEMU_VIRTIO_RNG: "0"
+      - kde_live_upgrade_leap_15.3:
```
Comment (grisu48) [inline job_groups/opensuse_leap_15.6_images.yaml:145]: Do we know why there are no 15.4 and 15.5 test runs? :thinking:

**Review by grisu48 — APPROVED** [review]
One question, otherwise LGTM!

File: job_groups/opensuse_leap_15.6_images.yaml (hunk 1 above)
Comment (Vogtinator) [reply job_groups/opensuse_leap_15.6_images.yaml:145]: I don't think the necessary HDD was ever created

Comment (grisu48) [reply job_groups/opensuse_leap_15.6_images.yaml:145]: Yep, that's the case

**Pull request merged**